src/components/yelp/yelpReviews.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box } from '@mui/material';
+import React from 'react';
+import { useEffect } from 'react';
+
+const reviewsIds = [
```

Comment:
Do we want/need this ids hard coded?

Reply:
we only want to pull the 5 star reviews. we can use an API call to fetch the data but then we would need to clean it up before each render. this is simple solution that should work for now.